### PR TITLE
feat: Add license banner

### LINKED ISSUE #1039: License banner in every source file
**Description**

Ideally to automate and use some tool to maintain that.

Banner text:
`// Use of this source code is governed by the Apache License, Version 2.0 that can be found in the LICENSE file.`

**Acceptance Criteria**
- [ ] Approve example file before applying banner with @illia-romanenko and @bootstraponline 
- [ ] After approval of example apply to all files
- [ ] Add Github Actions check for this license

**Additional context**
Example of similar banner: https://github.com/dart-lang/sdk/blob/master/sdk/lib/cli/wait_for.dart

### PR DESCRIPTION
**Pull request by solid-tetianamovlian**
Fixes #1039

### CODE REVIEW

**Review by solid-vladlaba — CHANGES_REQUESTED** [review]
Well done, @solid-tetianamovlian @solid-maxim! I've added a couple of suggestions below - please let me know what you think!

File: .licenserc.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "**/*.js": [
```
[the remark this answers is not in the document]
Comment (solid-tetianamovlian) [reply .licenserc.json:2]: As we’ve tested, multiple file extensions currently are not working.

[the remark this answers is not in the document]
Comment (solid-maxim) [reply .licenserc.json]: In the updated version of the License checker, this issue was resolved. So we've fixed this according to the suggestion.

**Review by solid-vladlaba — APPROVED** [review]
LGTM, thanks @solid-tetianamovlian @solid-maxim!:rocket:

**Pull request merged**